.github/workflows/assignments_test_docker.yml

```diff
@@ -21,12 +26,14 @@ jobs:
       - name: Build Docker image
         working-directory: ./weblab-docker
         env:
-          BUILD_ARGS: "--build-arg ANDY_BRANCH=${{ github.sha }}"
+          BUILD_ARGS: "--build-arg ANDY_BRANCH=${{ env.GHA_SHA }} --build-arg ANDY_REPO=${{ env.GHA_REPO }}"
         run: |
           make
 
       - name: Run the reference solutions of all assignments and verify the scores are 100/100
         env:
-          COMMIT_HASH: ${{ github.sha }}
+          # github.repository contains owner, we don't want that
+          REPO_NAME: ${{ github.event.repository.name }}
```

Comment:
Instead of using the repo name, could we perhaps set the `path` option of the checkout action to "andy" so that it is always "andy" even when the fork name is different?